app/backend/Program.cs

```diff
@@ -16,6 +19,20 @@
 builder.Services.AddAzureServices();
 builder.Services.AddAntiforgery(options => { options.HeaderName = "X-CSRF-TOKEN-HEADER"; options.FormFieldName = "X-CSRF-TOKEN-FORM"; });
 builder.Services.AddHttpClient();
+builder.Services.AddSignalR().AddAzureSignalR(options =>
+{
+    static string? GetEnvVar(string key) => Environment.GetEnvironmentVariable(key);
```

Comment:
Use the default way not reading from scratch? [Read from conf](https://learn.microsoft.com/en-us/azure/azure-signalr/signalr-howto-authorize-managed-identity#use-a-system-assigned-identity)